### ISSUE TITLE
Translate text on images to Spanish

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#2050]
There are a few images and GIFs that have English text on them. Replacing the text there wasn't as easy as replacing actual text, so I left that out of the initial internationalization. Whoever makes the new images should write out a process for doing it and put it in the wiki so that we can easily modify future images (and potentially add more languages).

MLGraphic.gif
![MLGraphic](https://user-images.githubusercontent.com/6518824/78303789-3ba81400-74f2-11ea-8f70-b7285ff642a8.gif)

Still need translations for this. Also am not confident that you can easily modify a GIF like this.

ExamplePictures02.png
![ExamplePictures02](https://user-images.githubusercontent.com/6518824/78303810-42cf2200-74f2-11ea-94c3-3655d35bd304.png)
Translations:
Accessible. Curb ramps help wheelchair users get on and off the sidewalks.
Accesible. Las rampas peatonales ayudan a las personas usuarias de sillas de ruedas a subir y bajar de las banquetas.

Inaccessible. Wheelchair users can't navigate sidewalks when curb ramps are missing.
Inaccesible. Las personas usuarias de sillas de ruedas no pueden circular por las banquetas cuando faltan las rampas.

Inaccessible. Obstacles block the paths of wheelchair users.
Inaccesible. Los obstáculos en la banqueta bloquean el camino de las personas usuarias de sillas de ruedas.

Inaccessible. Sidewalks with surface problems may not be navigable.
Inaccesible. Las banquetas con problemas en la superficie pueden no ser transitables.

ExamplePictures03_v2.png
![ExamplePictures03_v2](https://user-images.githubusercontent.com/6518824/78303816-482c6c80-74f2-11ea-9528-006124a89564.png)
Translations:
curb ramps
rampas peatonales

missing curb ramps
rampas peatonales ausentes

sidewalk obstacles
obstáculos en la banqueta

surface problems
problemas en superficie

DoubleClick.png
![DoubleClick](https://user-images.githubusercontent.com/6518824/78303700-14e9dd80-74f2-11ea-9770-de65ee98f569.png)
Translation:
Double click here
Haz doble clic aquí

**Comment by gari01234** [comment]
-Machine labels
Etiquetas automatizadas

-User labels
Etiquetas de usuarias

-Better city planning
Mejor planificación urbana 

-Better temporal tracking
Mejor seguimiento temporal

-Interactive visualization tools
Herramientas de visualización interactiva

-Smarter routing
Enrutamiento inteligente

-Machine learning
Aprendizaje de máquinas

-Outcomes
Resultados

https://docs.google.com/document/d/1nkBRJ2BDk6M6scgm596d6BeEYqHXb3VQeFgTMOOB-6o/edit?usp=sharing

**Comment by jstadeusiak** [comment]
**Guide Posted Here** https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Translating-Text-On-Images-Guide

Attached below are some examples I completed using the method in the guide. 
![DoubleClickHerePng](https://user-images.githubusercontent.com/64165755/86409176-2c5d7180-bc6d-11ea-9d33-fe6e1b16ff59.png) ![DoubleClickTranslated](https://user-images.githubusercontent.com/64165755/86409178-2e273500-bc6d-11ea-80e7-9d6a2dd10cca.png)

![ExamplePicture03](https://user-images.githubusercontent.com/64165755/86409337-8827fa80-bc6d-11ea-8031-c07870dc71c1.png)
![ExamplePicture03_v2Translated](https://user-images.githubusercontent.com/64165755/86409258-5747c580-bc6d-11ea-976a-08a3841577c8.png)

**Comment by misaugstad** [comment]
These look really good! For the double click here, it looks like the text on the Spanish version is a little bit too low.

And for the 2nd image, could you try reducing the font size a tiny bit to try and have it not spill over the sides of the images? I don't want to decrease the size too much, but it looks a bit messy like this. Also could you text to only have the first letter of each phrase capitalized (just for the Spanish version)? Spanish capitalization conventions are a bit different from US English.

As for the documentation, thank you for writing it!! In there, the first thing it says is that the recommended software is photoshop. Is it recommended for any particular reason? Or just because it's the only one you've tested it with?

**Comment by jstadeusiak** [comment]
Photoshop was the only software I personally used to do this. I know there are other equivalent software but Photoshop seems to be the most mainstream of all them, which is why I included it. I could add some Photoshop alternatives in the guide as well, but the specifics for the procedures may be a slightly different in those alternative apps.

**Comment by misaugstad** [comment]
I don't think you need to add alternatives, I would just change the wording to say something about how the guide was written for photoshop instead of saying that photoshop is recommended and ideal.

**Comment by jstadeusiak** [comment]
Updated:


![doubleclickfinal](https://user-images.githubusercontent.com/64165755/86834839-31953480-c050-11ea-887f-ef897d923c69.png)

![ExamplePicture03Translated](https://user-images.githubusercontent.com/64165755/86835465-0232f780-c051-11ea-9532-3c4879b2de27.png)

Still a little bit of spillover on the second image, I did not want to compromise the font size too much.

**Comment by misaugstad** [comment]
Pasting these here again bc it is easier to compare side by side. The click here now looks like it is too high. But I like your choice of font size for the other one. I think the tradeoff is appropriate.

![DoubleClickHerePng](https://user-images.githubusercontent.com/64165755/86409176-2c5d7180-bc6d-11ea-9d33-fe6e1b16ff59.png) ![DoubleClickTranslated](https://user-images.githubusercontent.com/64165755/86409178-2e273500-bc6d-11ea-80e7-9d6a2dd10cca.png) ![doubleclickfinal](https://user-images.githubusercontent.com/64165755/86834839-31953480-c050-11ea-887f-ef897d923c69.png)

![ExamplePicture03](https://user-images.githubusercontent.com/64165755/86409337-8827fa80-bc6d-11ea-8031-c07870dc71c1.png)
![ExamplePicture03_v2Translated](https://user-images.githubusercontent.com/64165755/86409258-5747c580-bc6d-11ea-976a-08a3841577c8.png)
![ExamplePicture03Translated](https://user-images.githubusercontent.com/64165755/86835465-0232f780-c051-11ea-9532-3c4879b2de27.png)

**Comment by jstadeusiak** [comment]
![86409176-2c5d7180-bc6d-11ea-9d33-fe6e1b16ff59](https://user-images.githubusercontent.com/64165755/86962324-8c459380-c117-11ea-9d63-d8f5bbe4054d.png) ![revised png](https://user-images.githubusercontent.com/64165755/86962417-b7c87e00-c117-11ea-8177-c24b30bd1692.png)

Found the balance for height on these.

**Comment by misaugstad** [comment]
That looks better, thanks!

**Comment by misaugstad** [comment]
linking to #2253

**Comment by misaugstad** [comment]
The only thing we have left to translate is the MLGraphic.gif file on the landing page, which we hadn't translated before because it is more difficult to do with a GIF. I did manage to remove the text from the GIF using https://ezgif.com/ with their "censor" tool to cover up the text. Now we should be able to overlay translations on top of the GIF using CSS, and can thus swap in translations easily.

Though we might want to consider further editing to include our new label type design, since we're messing with it anyway?

Here's the current state of the GIF, with the text removed:
![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/6518824/159349887-9ff48c50-7b5f-46cf-9a77-ad706a41e622.gif)

And here are the Spanish translations from above, provided by @gari01234 
- Machine labels: Etiquetas automatizadas
- User labels: Etiquetas de usuarias
- Better city planning: Mejor planificación urbana
- Better temporal tracking: Mejor seguimiento temporal
- Interactive visualization tools: Herramientas de visualización interactiva
- Smarter routing: Enrutamiento inteligente
- Machine learning: Aprendizaje de máquinas
- Outcomes: Resultados

**Comment by misaugstad** [comment]
@Z-Setiadi a couple things to note when making this change:

* Just use Google Translate for the Dutch translations for now. I'll record these as translations that we'll need, and once we get a large batch of them, I'll send them along for someone to give us more permanent translations :)
* Make sure to test the spacing and how it affects the rest of the landing page. Check with various zoom levels in multiple language to make sure that any CSS changes don't shift around other elements on the page
